CMakeLists.txt

```diff
@@ -119,6 +119,26 @@ execute_process(
   OUTPUT_VARIABLE hipcc_version_string)
 message(STATUS "hipcc version:    ${hipcc_version_string}")
 
+## Check for ROCm version
+execute_process(
+  COMMAND         bash "-c" "cat ${ROCM_PATH}/.info/version"
+  OUTPUT_VARIABLE rocm_version_string
+)
+string(REGEX MATCH "([0-9]+)\\.([0-9]+)\\.([0-9]+)" rocm_version_matches ${rocm_version_string})
+if (rocm_version_matches)
+    set(ROCM_MAJOR_VERSION ${CMAKE_MATCH_1})
+    set(ROCM_MINOR_VERSION ${CMAKE_MATCH_2})
+    set(ROCM_PATCH_VERSION ${CMAKE_MATCH_3})
+
+    message(STATUS "ROCm version: ${ROCM_MAJOR_VERSION}.${ROCM_MINOR_VERSION}.${ROCM_PATCH_VERSION}")
```

Comment:
are you ignoring the digits after `-` like in `6.0.0-12568`?

Reply:
yes, I think the digits after '-' are not important in this case